diff -Naurp squashfs-3.3.orig/squashfs-tools/mksquashfs.c squashfs-3.3/squashfs-tools/mksquashfs.c
--- squashfs-3.3.orig/squashfs-tools/mksquashfs.c	2008-03-29 08:38:43.000000000 +0000
+++ squashfs-3.3/squashfs-tools/mksquashfs.c	2008-03-29 08:38:57.000000000 +0000
@@ -3503,7 +3503,7 @@ printOptions:
 		s_minor = SQUASHFS_MINOR;
 
         for(i = 0; i < source; i++)
-                if(stat(source_path[i], &source_buf) == -1) {
+                if(lstat(source_path[i], &source_buf) == -1) {
                         fprintf(stderr, "Cannot stat source directory \"%s\" because %s\n", source_path[i], strerror(errno));
                         EXIT_MKSQUASHFS();
                 }
