diff -Naurp squashfs.orig/linux-2.6/inode.c squashfs/linux-2.6/inode.c
--- squashfs.orig/linux-2.6/inode.c	2008-03-21 19:50:35.000000000 +0000
+++ squashfs/linux-2.6/inode.c	2008-03-21 19:52:47.000000000 +0000
@@ -21,12 +21,12 @@
  * inode.c
  */
 
-#include <linux/squashfs_fs.h>
+#include "squashfs_fs.h"
 #include <linux/module.h>
 #include <linux/zlib.h>
 #include <linux/fs.h>
-#include <linux/squashfs_fs_sb.h>
-#include <linux/squashfs_fs_i.h>
+#include "squashfs_fs_sb.h"
+#include "squashfs_fs_i.h"
 #include <linux/buffer_head.h>
 #include <linux/vfs.h>
 #include <linux/vmalloc.h>
diff -Naurp squashfs.orig/linux-2.6/squashfs2_0.c squashfs/linux-2.6/squashfs2_0.c
--- squashfs.orig/linux-2.6/squashfs2_0.c	2008-03-21 19:49:30.000000000 +0000
+++ squashfs/linux-2.6/squashfs2_0.c	2008-03-21 19:52:47.000000000 +0000
@@ -21,12 +21,12 @@
  * squashfs2_0.c
  */
 
-#include <linux/squashfs_fs.h>
+#include "squashfs_fs.h"
 #include <linux/module.h>
 #include <linux/zlib.h>
 #include <linux/fs.h>
-#include <linux/squashfs_fs_sb.h>
-#include <linux/squashfs_fs_i.h>
+#include "squashfs_fs_sb.h"
+#include "squashfs_fs_i.h"
 
 #include "squashfs.h"
 static int squashfs_readdir_2(struct file *file, void *dirent, filldir_t filldir);
diff -Naurp squashfs.orig/linux-2.6/squashfs_fs_sb.h squashfs/linux-2.6/squashfs_fs_sb.h
--- squashfs.orig/linux-2.6/squashfs_fs_sb.h	2008-03-21 19:49:30.000000000 +0000
+++ squashfs/linux-2.6/squashfs_fs_sb.h	2008-03-21 19:52:47.000000000 +0000
@@ -23,7 +23,7 @@
  * squashfs_fs_sb.h
  */
 
-#include <linux/squashfs_fs.h>
+#include "squashfs_fs.h"
 
 struct squashfs_cache {
 	long long	block;
