diff -Naurp squashfs.orig/linux-2.6/Makefile squashfs/linux-2.6/Makefile
--- squashfs.orig/linux-2.6/Makefile	2008-03-21 19:49:30.000000000 +0000
+++ squashfs/linux-2.6/Makefile	2008-03-21 19:54:59.000000000 +0000
@@ -2,6 +2,16 @@
 # Makefile for the linux squashfs routines.
 #
 
+CONFIG_SQUASHFS=m
+
+EXTRA_CFLAGS := -DCONFIG_SQUASHFS_FRAGMENT_CACHE_SIZE=3 -DUnsquashNoPreempt
+
 obj-$(CONFIG_SQUASHFS) += squashfs.o
 squashfs-y += inode.o
 squashfs-y += squashfs2_0.o
+
+all:
+	$(MAKE) -C /lib/modules/$(shell uname -r)/build M=$(PWD) modules EXTRA_CFLAGS="$(EXTRA_CFLAGS)"
+
+clean:
+	$(MAKE) -C /lib/modules/$(shell uname -r)/build M=$(PWD) clean
